Bottom: 5a5c07a255b18f25a870ce0153c55208ad22d67d
Top:    3451785850d3447607670ba07518f2418ec14ccc
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-16 09:17:44 -0500

Add new dependency, bc, to Dockerfile


---
diff --git a/docker/buildbot-worker-ptxdist/Dockerfile b/docker/buildbot-worker-ptxdist/Dockerfile
index ee6bfeb5..b14d49c4 100644
--- a/docker/buildbot-worker-ptxdist/Dockerfile
+++ b/docker/buildbot-worker-ptxdist/Dockerfile
@@ -22,6 +22,7 @@ RUN apt-get update \
 &&  DEBIAN_FRONTEND=noninteractive \
     apt-get -y install -q \
         autoconf \
+        bc \
         bison \
         build-essential \
         ccache \
